.distignore

```diff
@@ -2,39 +2,37 @@
 .git
 .github
 /bin
-node_modules
+/node_modules
```

Comment:
@circlecube Maybe we should keep this without the leading slash? I think it's pretty safe to exclude this globally. Otherwise, we need to add a `/vendor/newfold-labs/*/node_modules` exclusion.

Reply:
Updated